src/OSPSuite.Infrastructure.Import/Core/DataFormat/AbstractColumnsDataFormat.cs

```diff
@@ -325,7 +326,7 @@ private SimulationPoint parseMappingOnSameColumn(MappingDataFormatParameter curr
             if (!string.IsNullOrEmpty(currentParameter.MappedColumn.Unit.ColumnName))
                unitColumnDescription = data.GetColumnDescription(currentParameter.MappedColumn.Unit.ColumnName);
             if (unitColumnDescription == null)
-               throw new MissingColumnException(currentParameter.MappedColumn.Unit.ColumnName);
+               throw new MissingColumnException(new List<string>() { currentParameter.MappedColumn.Unit.ColumnName });
```

Comment:
Snce you own this exception, create an overload

Reply:
done